tools/release_builder.py

```diff
@@ -90,78 +83,53 @@ def __init__(
                 artifact_package_name = artifact_package_name[len('beta-'):]
             release_dir_path = artifact_package_name + '/assets'
 
-        # automatically include source universe URL in commit description:
-        if commit_desc:
-            self._commit_desc = '{}\n\nSource URL: {}'.format(commit_desc.rstrip('\n'), self._stub_universe_url)
-        else:
-            self._commit_desc = 'Source URL: {}'.format(self._stub_universe_url)
-
-        self._pr_title = 'Release {} {} (automated commit)\n\n'.format(
-            self._pkg_name, self._pkg_version)
-        self._release_artifact_http_dir = '{}/{}/{}'.format(
-            http_release_server, release_dir_path, self._pkg_version)
-        self._release_artifact_s3_dir = 's3://{}/{}/{}'.format(
+        s3_directory_url = 's3://{}/{}/{}'.format(
```

Comment:
These are the same as in the `AWSPublisher`, correct? Does it make sense to unify them somehow?

Reply:
OK. That's fine for now then.